protocol/tags.go

```diff
@@ -37,23 +37,7 @@ const (
 	TxnTag             Tag = "TX"
 	UniCatchupReqTag   Tag = "UC"
 	UniEnsBlockReqTag  Tag = "UE"
-	UniEnsBlockResTag  Tag = "US"
-	UniCatchupResTag   Tag = "UT"
+	UniEnsBlockResTag  Tag = "US" // was used for wsfetcherservice
```

Comment:
I would comment out these tags to ensure we don't use these.

Reply:
Also, you would need to eliminate the use of UniCatchupReqTag/UniEnsBlockReqTag, but it's ok to defer that.